src/functions/updateDmsTaskWithDates/framework/task-modifier.ts

```diff
@@ -0,0 +1,87 @@
+import { getLogger } from './util';
+import { DmsApi } from './dms/dms';
+import { addBetweenFilter, addOnOrAfterFilter, addOnOrBeforeFilter, Options } from './dms/table-mapping';
+import { DateTime, Duration } from 'luxon';
```

Comment:
Did moment.js not work? Would be nice to be consistent and use moment for dates/times everywhere.

Reply:
Was just lifting and shifting what was in the cli-app. Can replace with moment.

Reply:
Just saw that luxon is published by moment so perhaps there's no need, would be useful to know what the differences are with the main moment lib though